consensus/src/main/java/org/ethereum/beacon/consensus/SpecHelpers.java

```diff
@@ -593,44 +725,30 @@ def activate_validator(state: BeaconState, index: int, genesis: bool) -> None:
       )
    */
   public void activate_validator(MutableBeaconState state, ValidatorIndex index, boolean genesis) {
-    SlotNumber activationSlot = genesis ?
-        spec.getGenesisSlot() :
-        state.getSlot().plus(spec.getEntryExitDelay());
+    EpochNumber activationSlot = genesis ?
```

Comment:
It's not in google style as I remember it

Reply:
What exactly violated here?

Reply:
```suggestion
       EpochNumber activationSlot =
        genesis ? spec.getGenesisEpoch() : get_entry_exit_effect_epoch(get_current_epoch(state));
```
or 
```suggestion
        EpochNumber activationSlot = genesis 
        ? spec.getGenesisEpoch() 
        : get_entry_exit_effect_epoch(get_current_epoch(state));
```

by default it suggests first as it fits in line. If you make it longer it will suggest 2nd